README.md

```diff
@@ -119,6 +112,77 @@ Click the button below to start deployment.
    
 If you would like to use other parameters please change respective variable values in the the template or contact Alert Logic.
 
+
+### Event Hub Filtering
+
+**a) Event Hub Filter Regex** - Type the filter in REGEX format.
+   **example:** \/*.Policy or "Policy"
+
+   **Note:** For "Event Hub Filter Json" and "Event Hub Filter Regex", only messages which contain the specified property will be collected. If both the filter values are provided then logs will be collected based on both the values.
+
+   **b) Event Hub Filter Json** - Type the filter in JSON format.
+   **example json log:** 
+   <pre><code>
+   log = [
+      {
+         "resultType1":"Success1",
+         "type":"result",
+         "user":"user1"
+      },
+      {
+         "resultType2": {
+            "status":"Success2",
+            "type":"result",
+            "user":"user2"
+         }
+      },
+      {
+         "resultType3": {
+            "status": {
+               "result":"Success3"
+            },
+            "type": {
+               "value": "result"
+            },
+            "user": {
+               "value": "user3"
+            }
+         }
+      }
+   ]
+   </code></pre>
+   ***Root level filtering example:*** 
```

Comment:
please put all sample into one table. Also JSOn formatting is not correct.
|Description|Filter|Output|
|--|--|--|
|||

Reply:
Done